src/providers/fault-count/__tests__/fault-count.spec.ts

```diff
@@ -256,66 +281,6 @@ describe('FaultCountProvider', () => {
         expect((FaultCountDHelper as any).getDangerousFaultSumCountCatD1E).toHaveBeenCalled();
       });
     });
-    it('should call the category AM1 specific method for getting the dangerous fault sum count', () => {
-      faultCountProvider.getDangerousFaultSumCount(TestCategory.EUAM1, catAM1TestDataStateObject);
-      expect((FaultCountAM1Helper as any).getDangerousFaultSumCountCatAM1).toHaveBeenCalled();
-    });
-  });
-
-  describe('getDrivingFaultSumCount', () => {
```

Comment:
What's the reason for deleting all these tests?

Reply:
They are duplicated in the same file